persistence/README.md

```diff
@@ -78,232 +114,306 @@ This document presents a specification for the Persistence Module of the Pocket
 
-
 ### 3.1. Overview
 
-The **state dataset** contains the result of each set of state transitions indicated by the transactions of any given height. This makes **immutability** of the **state dataset** a desired property, because the resulting state at any given height is one of the most important inputs of the consensus process of the Pocket Network. This specification contains a mechanism that leverages [Merkle Patricia Tries](https://eth.wiki/en/fundamentals/patricia-tree) as a data-structure that helps prove the integrity of the state dataset at any given height, sans 1 modification, the change of the RLP encoding scheme, for the [Protobuf](https://developers.google.com/protocol-buffers) encoding scheme. 
-
+The **state dataset** contains the result of each set of state transitions indicated by the transactions for a given height. This provides the **state dataset** the desired property of **immutability** because the resulting state at any given height is one of the most critical inputs of the consensus process of the Pocket Network. Furthermore, this specification contains a mechanism that leverages [Merkle Patricia Tries](https://eth.wiki/en/fundamentals/patricia-tree) as a data structure that helps prove the integrity of the state dataset at any given height, free of any modifications, including the selected encoding scheme (e.g. Protobuf](https://developers.google.com/protocol-buffers)).
 
 ### 3.2. Immutable State Schema
 
-We will refer to the **state dataset** as a group of collections that will conform to the immutable state schema. Every structure in the collection will become a leaf in a Merkle Patricia Trie, yielding a Merkle Root. Every merkle root in the dataset will be concatenated in lexicographical order based on the collection identifier and hashed using **SHA256** function to produce what we will refer to as the **state hash**.
-
-Every computed state hash will be persisted, alongside a signature computed from a designated private key, which will serve as a local proof of state computation. This proof will be utilized to avoid recalculation of previous states, and will allow the system, provided the same private key, will have a high degree of confidence in their computed state. In the case of any proof of tampering, state can be recalculated using the state hash algorithm. This proof will be called a **state computation proof.**
+We will refer to the **state dataset** as a group of collections that will conform to the immutable state schema. Every structure in the collection will become a leaf in a Merkle Patricia Trie, yielding a Merkle Root. Every Merkle root in the dataset will be concatenated in lexicographical order based on the collection identifier and hashed using **SHA256** function to produce what we will refer to as the **state hash**.
 
+<!-- TODO(discuss in review): Should we add a diagram for this now or later to make it easier to understand how the data is stored -->
```

Comment:
Yes, I think a diagram explaining how a CRUD operation to the mutable state schema yields an equivalent operation in the immutable state schema.

Reply:
I changed `TODO(discuss in review)` to `TODO(olshansky)` so I can tend to it in the next 1-2 weeks and we can submit these initial changes as is. 👍 ?